classes/fieldset/field.php

```diff
@@ -602,8 +609,13 @@ protected function template($build_field)
 				return $template;
 			}
 
-			// still here? wasn't a multi field template available, try the normal one with imploded $build_field
-			$build_field = implode(' ', $build_field);
+            // map the key value $build_field (to have <input...>label)
+            $labels = array_keys($build_field);
+            $buildFieldMap = array_map(function($value, $key) {
```

Comment:
When radio/checkbox field is multiple like :
```
'form' => array(
                'type' => 'radio',
                'value' => 'journaliste',
                'options' => array(
                    'journaliste' => __('Journaliste'),
                    'pro' => __('Professionnel'),
                ),
            ),
```
'$build_field' is array with `'label display' => 'value input display'`
But there was no label for each input display.